<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-SGH30GM" name="com.Test.qa.Screenshot" tests="1" failures="0" timestamp="21 Feb 2018 21:13:00 GMT" time="0.015" errors="1">
  <testcase name="screenshot" time="0.015" classname="com.Test.qa.Screenshot">
    <error type="java.lang.Error" message="Unresolved compilation problems: 
	Type mismatch: cannot convert from java.io.File to com.gargoylesoftware.htmlunit.javascript.host.file.File
	The method copyFile(File, File) is undefined for the type FileUtils
	The constructor File(String) is not visible
">
      <![CDATA[java.lang.Error: Unresolved compilation problems: Type mismatch: cannot convert from java.io.File to com.gargoylesoftware.htmlunit.javascript.host.file.FileThe method copyFile(File, File) is undefined for the type FileUtilsThe constructor File(String) is not visibleat com.Test.qa.Screenshot.screenshot(Screenshot.java:29)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)at java.lang.reflect.Method.invoke(Unknown Source)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)]]>
    </error>
  </testcase> <!-- screenshot -->
</testsuite> <!-- com.Test.qa.Screenshot -->
